docs/en-US/component/collapse.md

```diff
@@ -62,15 +72,17 @@ collapse/customization
 
 ### Collapse Item Attributes
 
-| Name     | Description                        | Type                  | Default |
-| -------- | ---------------------------------- | --------------------- | ------- |
-| name     | unique identification of the panel | ^[string] / ^[number] | —       |
-| title    | title of the panel                 | ^[string]             | ''      |
-| disabled | disable the collapse item          | ^[boolean]            | false   |
+| Name          | Description                        | Type                     | Default |
+| ------------- | ---------------------------------- | ------------------------ | ------- |
+| name          | unique identification of the panel | ^[string] / ^[number]    | —       |
+| title         | title of the panel                 | ^[string]                | ''      |
+| icon ^(2.8.3) | icon of the collapse item          | ^[string] / ^[Component] | -       |
```

Comment:
It is best to mark it in the document.

![image](https://github.com/user-attachments/assets/49c11f03-8110-48f0-bb88-845d3cfef884)